packages/connect-examples/new-example/app/components/device/DeviceConnector.tsx

```
@@ -0,0 +1,495 @@
+    try {
+      console.log("[DeviceConnector] 🔗 连接设备:", device);
+      const sdk = await getSDKInstance();
+      const result = await sdk.getFeatures(device.connectId!);
+
+      if (result.success) {
+        const features = result.payload;
+        setDeviceFeatures(features);
+
+        // 从Features中获取正确的设备类型
+        const actualDeviceType = getDeviceType(features);
+        console.log("[DeviceConnector] 📋 设备类型检测结果:");
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)